src/components/file-browser/FilesTable.ts

```diff
@@ -24,7 +24,6 @@ const FilesTable = (props: FilesTableProps) => {
 
   return div({ style: { display: 'flex', flex: '1 1 auto' } }, [
     h(AutoSizer, {}, [
-      // @ts-expect-error
```

Comment:
Why is this change necessary?

Reply:
Adding the types module changed the types so that that this line was no longer necessary, so I had to remove it.